src/status_im/ui2/screens/chat/group_details/style.cljs

```diff
@@ -18,3 +18,23 @@
    :height           16
    :border-radius    6
    :background-color (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5)})
+
+(defn close-icon []
+  {:background-color (colors/theme-colors colors/neutral-10 colors/neutral-80)
+   :margin-left      20
+   :width            32
+   :height           32
+   :border-radius    10
+   :justify-content  :center
+   :align-items      :center
+   :margin-bottom    24})
+
+(defn bottom-container [safe-area]
+  {:position           :absolute
```

Comment:
what's the point of using :absolute ?

Reply:
Because its position is absolute

Reply:
yeah just wondering whats the reason, why its not just regular flex layout